.gitignore

```diff
@@ -44,3 +44,6 @@ atlassian-ide-plugin.xml
 # aet-vagrant
 /vagrant/.vagrant
 /vagrant/Berksfile.lock
+misc/deployer/.gradle/
+misc/deployer/gradle.properties
```

Comment:
Could we add sample file (`gradle.sample.properties`) to make it obvious what config parameters are expected to be in `gradle.properties` file?